submissions/Vlada-khom/html-css-popup/styles.css

```diff
@@ -0,0 +1,185 @@
+body {
+  background-color: rgba(0, 255, 255, 0.2);
+}
+
+header nav {
+  text-align: right;
+  font-family: "Roboto";
```

Comment:
Always add fallback font. [Read more about it](https://css-tricks.com/css-basics-fallback-font-stacks-robust-web-typography/)